system_tests/pubsub.py

```diff
@@ -203,21 +217,24 @@ def test_subscription_iam_policy(self):
         topic_name = 'test-sub-iam-policy-topic' + unique_resource_id('-')
         topic = Config.CLIENT.topic(topic_name)
         topic.create()
-        count = 5
-        while count > 0 and not topic.exists():
-            time.sleep(1)
-            count -= 1
+
+        # Retry / backoff up to 7 seconds (1 + 2 + 4)
+        retry = RetryResult(lambda result: result, max_tries=4)
```

Comment:
Isn't 4 the default?

In the same breath, shouldn't the do-nothing method be the default?